tests/tests/test_mender_connect.py

```diff
@@ -180,3 +181,70 @@ def test_bogus_proto_message(
             assert prot.props["status"] == protomsg.PROP_STATUS_ERROR
             assert prot.protoType == 12345
             assert prot.typ == proto_shell.MSG_TYPE_SPAWN_SHELL
+
+    def test_session_recording(
+        self, class_persistent_standard_setup_one_client_bootstrapped
+    ):
+        def get_cmd(ws, timeout=1):
```

Comment:
I think `get_cmd()` does the same thing as the already existing `shell.recvOutput()`, doesn't it? Get output with a small wait?

Reply:
The only difference is that it filters all buffered messages by message type (`== command`) instead of making assertions.